app/lib/matador/components/job-info/JobInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import { Grid, MantineColor } from "@mantine/core";
+import { StatCard } from "../stat-card";
+
+interface JobInfoProps {
```

Comment:
export the type

Reply:
Fixed with 35c6421357701641e9eedeec5819dece439e234c